src/hotspot/share/compiler/compilerDirectives.hpp

```diff
@@ -128,6 +128,8 @@ class DirectiveSet : public CHeapObj<mtCompiler> {
   bool is_intrinsic_disabled(const methodHandle& method);
   static ccstrlist canonicalize_control_intrinsic(ccstrlist option_value);
   void finalize(outputStream* st);
+  bool is_c1(CompilerDirectives* directive);
```

Comment:
NIT: these methods can be made "const".

Reply:
@JohnTortugo 
Thank you for your advice.
I added const to these methods.